base_settings.py

```
@@ -0,0 +1,53 @@
+]
+
+TIME_ZONE = "Europe/Berlin"
+
+STATIC_URL = "/static/"
+
+# These are commonly used in both, but can be overridden if needed
+ROOT_URLCONF = "endoreg_db.urls"
+
+# Import paths and storage helpers for use in child settings
+
+from endoreg_db.logger_conf import get_logging_config
```

Ruff (0.11.9)

`endoreg_db.logger_conf.get_logging_config` imported but unused

Remove unused import: `endoreg_db.logger_conf.get_logging_config`

(F401)